AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -101,16 +218,25 @@ def disable_component(self):
         """
         editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [self.id])
 
+    def enable_component(self):
+        """
+        used to enable the componet using its id value
+        """
+        editor.EditorComponentAPIBus(bus.Broadcast, "EnabledComponents", [self.id])
+
     @staticmethod
-    def get_type_ids(component_names: list) -> list:
+    def get_type_ids(component_names: list, entity_type: str ='Game') -> list:
```

Comment:
use a enum instead of a string for this:
https://docs.python.org/3/library/enum.html

Like:

    class EntityType(Enum):
        GAME = azlmbr.entity.EntityType().Game
        LEVEL = azlmbr.entity.EntityType().Level

This will also avoid having to manually convert it to azlmbr

Reply:
fixed